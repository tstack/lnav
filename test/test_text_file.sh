#! /bin/bash

export YES_COLOR=1
unset XDG_CONFIG_HOME

run_cap_test ${lnav_test} -n \
    ${top_srcdir}/README.md

run_cap_test ${lnav_test} -n -c ':goto #screenshot' \
    ${top_srcdir}/README.md

run_cap_test ${lnav_test} -n ${top_srcdir}/README.md#screenshot

# run_cap_test ${lnav_test} -n ${test_dir}/non-existent:4

run_cap_test ${lnav_test} -n ${top_srcdir}/README.md:-4

run_cap_test ${lnav_test} -n \
    -c ':goto 115' \
    -c ";SELECT top_meta FROM lnav_views WHERE name = 'text'" \
    -c ':write-json-to -' \
    ${top_srcdir}/README.md

run_cap_test ${lnav_test} -n \
    ${top_srcdir}/src/log_level.cc

cp ${test_dir}/UTF-8-test.txt UTF-8-test.md
run_cap_test ${lnav_test} -n \
    UTF-8-test.md

run_cap_test ${lnav_test} -n \
    -c ';SELECT * FROM lnav_file_metadata' \
    ${test_dir}/textfile_0.md
