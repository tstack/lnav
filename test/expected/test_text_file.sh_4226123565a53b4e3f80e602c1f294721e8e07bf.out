[32m# Test file for syntax-highlighting[0m

[1m[36mdef[0m hello(abc):
    [35m"""[0m
[35m    This is a multi-line string[0m

[35m    [0m[35m@param[0m[35m abc: blah blah[0m
[35m    """[0m
    [1m[36mreturn[0m abc + 1
